source/opt/basic_block.h

```diff
@@ -45,7 +45,7 @@ class BasicBlock {
   // Appends an instruction to this basic block.
   inline void AddInstruction(std::unique_ptr<Instruction> i);
   // The label starting this basic block.
-  Instruction& Label() { return *label_; }
+  Instruction* GetLabelInst() { return &*label_; }
```

Comment:
I can't find any uses of this method.   Can we eliminate this altogether?
(Also, why would returning the pointer be preferable? Since it's always non-null isn't it better to return a reference?)